### PR TITLE
fix: text visibility in dark mode

### LINKED ISSUE #1406: text-visibility-in-dark-mode
### Description

https://meshery.io/blog/mesheryctl-system-provider : 
A dark-mode bug related to text-visibility 
Screenshot 2023-09-04 185601

### Expected Behavior

Text should be visible in both light as well as dark-mode.

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by OmAximani0**
- Fixes: #1406 

**Description**
Previously text was not visible in dark mode, because element `<ol>` was not having specific `color` property

This PR fixes #1406

**Notes for Reviewers**
- I have also added `font-size` according to `<p>`
- Screenshot
image

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Mohith234** [comment]
@OmAximani0 The build preview is failing. Do check the logs and Fix it

**Comment by OmAximani0** [comment]
> @OmAximani0 The build preview is failing. Do check the logs and Fix it

Hey @Mohith234 , The preview is now live. Please have a review

File: _sass/elements.scss [hunk 1]
```diff
@@ -101,6 +101,11 @@ p, address {
 	line-height: 1.4em;
 }
 
+ol {
+	color: var(--color-grey-light);
```
Comment (vishalvivekm) [inline _sass/elements.scss:105]:
```suggestion
	color: var(--color-secondary-dark);
```

File: _sass/elements.scss [hunk 2]
```diff
@@ -101,6 +101,11 @@ p, address {
 	line-height: 1.4em;
 }
 
+ol {
+	color: var(--color-secondary-dark);
+	font-size: 1.1rem;
```
Comment (vishalvivekm) [inline _sass/elements.scss:106]:
```suggestion
```

**Review by vishalvivekm — APPROVED** [review]
Thanks @OmAximani0 🚀

**Comment by akhilender-bongirwar** [comment]
Hey @Mohith234 , @vishalvivekm , @OmAximani0 

In this PR , although the text looks consistent for the dark mode but the text does not look consistent in the light mode.

I have attached the screenshots of mesheryctl-system-provider blog.

Screenshot 2023-09-10 001854
Screenshot 2023-09-10 001916

**Pull request merged**